samples/SqlExtensionSamples/OutputBindingSamples/TimerTriggerProducts.cs

```diff
@@ -8,24 +8,28 @@
 using Microsoft.Extensions.Logging;
 using static SqlExtensionSamples.ProductUtilities;
 
-namespace SqlExtensionSamples.TriggerBindingSamples
+namespace SqlExtensionSamples
 {
     public static class TimerTriggerProducts
     {
         private static int _executionNumber = 0;
+
+        /// <summary>
+        /// This timer function runs every 5 seconds. Each time it generates 1000 rows of data, starting at a ProductID of 10000.
```

Comment:
Do we need to start with the product ID at 10000 anymore?

Reply:
We don't any more, forgot to update comment